src/services/riskFactor.js

```diff
@@ -14,9 +14,26 @@ export const calculateRiskFactor = (bac, drinkCountTimestamps, height, weight, i
     drinkCount++;
   }
 
+  const bac_lb = 0;
+  const bac_ub = 0.25;
+  const bac_new_ub = 100;
+  const drinkCount_lb = 0;
+  const drinkCount_ub = 6;
+  const drinkCount_new_ub = 100;
```

Comment:
These switched cases I think compared to how variable names are in different parts of code